$NetBSD: patch-ag,v 1.4 2022/04/24 10:57:28 tnn Exp $

Handle png.
Use raw instead of AIFF for ... (unclear what) ...

--- fxtv_cnvt.sh.orig	Mon Feb 12 22:18:40 2001
+++ fxtv_cnvt.sh
@@ -32,7 +32,7 @@
 #VIDEO_TARGET_FPS="30"
 #VIDEO_STREAM="YES"
 #
-#IMAGE_TARGET_FMT="YUV"                 # TIFF|PPM|YUV
+#IMAGE_TARGET_FMT="YUV"                 # TIFF|PNG|PPM|YUV
 #
 #CLEANUP_TEMP_FILES="YES"
 
@@ -73,6 +73,10 @@ MPEG_ENC_PARAM_STR_TIFF=\
 'BASE_FILE_FORMAT  PPM
 INPUT_CONVERT     tifftopnm *'
 
+MPEG_ENC_PARAM_STR_PNG=\
+'BASE_FILE_FORMAT  PPM
+INPUT_CONVERT     pngtopnm *'
+
 MPEG_ENC_PARAM_STR_PPM=\
 'BASE_FILE_FORMAT  PPM         
 INPUT_CONVERT     *'
@@ -181,8 +185,8 @@ GenAudioPipe() {
 
    # Do we need an intermediate file?
    if [ $mpeg_enc = YES ]; then
-      out_soxfn="$out_fnbase.aiff"
-      out_soxfmt="AIFF"
+      out_soxfn="$out_fnbase.raw"
+      out_soxfmt="RAW"
    else
       out_soxfn="$GAout_fn"
       out_soxfmt="$out_fmt"
@@ -258,6 +262,7 @@ WriteMPEGEncodeParamFile() {
     # Write the frame-format specific part 
     case $frame_fmt in
        TIFF)  str="$MPEG_ENC_PARAM_STR_TIFF";;
+       PNG)   str="$MPEG_ENC_PARAM_STR_PNG";;
        PPM)   str="$MPEG_ENC_PARAM_STR_PPM" ;;
        YUV)   str="$MPEG_ENC_PARAM_STR_IYUV";;
     esac
@@ -277,6 +282,7 @@ WriteMPEGEncodeParamFile() {
        base=`basename $out_fnbase`
        case $frame_fmt in
           TIFF)  ext="tif";;
+          PNG)   ext="png";;
           PPM)   ext="ppm";;
           YUV)   ext="yuv";;
        esac
@@ -321,6 +327,7 @@ CountFrameFiles() {
 
     case $frame_fmt in
        TIFF)  ext="tif";;
+       PNG)   ext="png";;
        PPM)   ext="ppm";;
        YUV)   ext="yuv";;
     esac
